mods/ra/rules/misc.yaml

```diff
@@ -352,7 +349,7 @@ powerproxy.sonarpulse:
 		Icon: sonar
 		Description: Sonar Pulse
 		LongDesc: Reveals all submarines in the vicinity for a \nshort time.
-		ChargeTime: 30
+		ChargeTime: 20
```

Comment:
Oh wait, true - this line should be reverted, sonar pulse was good as-is in terms of recharge.